client/blocks/reader-post-actions/index.jsx

```diff
@@ -20,7 +20,7 @@ import * as stats from 'reader/stats';
 import { localize } from 'i18n-calypso';
 import ExternalLink from 'components/external-link';
 
-const ReaderPostActions = ( { translate, post, site, onCommentClick, showEdit, showVisit, showMenu, iconSize, className } ) => {
+const ReaderPostActions = ( { translate, post, site, onCommentClick, showEdit, showVisit, showMenu, showMenuFollow, iconSize, className } ) => {
```

Comment:
I'm not sure what'll look best here but it might be time to break this list out and possibly just do:

```js
const ReaderPostActions = ( props ) => {
    const {
        translate, post, site, onCommentClick, showEdit,
        showVisit, showMenu, showMenuFollow, iconSize,
        className
    } = props;
```